apps/client/src/components/Layouts/DefaultLayout.tsx

```diff
@@ -1,18 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { useHistory } from "react-router-dom";
-import { useMatomo } from "@datapunt/matomo-tracker-react";
 import BaseLayout from "./BaseLayout";
+import withTracking from "../../hoc/withTracking";
 
-function DefaultLayout(props: { heading: string; children: React.ReactNode }) {
+interface DefaultLayoutProps {
+  heading: string;
+  children: React.ReactNode;
+  MatomoPageView: Function;
+}
+
+function DefaultLayout({
+  heading,
+  children,
+  MatomoPageView,
+}: DefaultLayoutProps) {
   const { location } = useHistory();
-  const { trackPageView } = useMatomo();
 
-  React.useEffect(() => {
-    // @datapunt Track Page View https://github.com/Amsterdam/matomo-tracker/tree/master/packages/react
-    trackPageView({});
-  }, [location, trackPageView]);
+  useEffect(() => {
+    MatomoPageView();
+  }, [MatomoPageView, location]);
```

Comment:
`MatomoPageView` can be removed from the `useEffect`

Reply:
My IDE won't allow me to, everything used in useEffect must be there

Reply:
okay, than not